config/crd/bases/ocs.openshift.io_storageclusters.yaml

```diff
@@ -349,6 +349,26 @@ spec:
                   enable:
                     type: boolean
                 type: object
+              backingStorageClasses:
+                description: BackingStorageClasses is a list of storageclasses that
+                  will be used for the storageDeviceSets.
+                items:
+                  description: BackingStorageClass defines the backing storageclass
+                    for StorageDeviceSet
+                  properties:
+                    metadata:
+                      type: object
+                    parameters:
```

Comment:
should it be an array?

Reply:
its a map same as rook CR which will helps us to pass the parameters directly when creating rook CR's